crypto/evp/evp_fetch.c

```diff
@@ -214,23 +240,24 @@ inner_evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
     OSSL_NAMEMAP *namemap = ossl_namemap_stored(libctx);
     uint32_t meth_id = 0;
     void *method = NULL;
+    int unsupported = 0;
 
     if (store == NULL || namemap == NULL)
-        return NULL;
+        goto end;
```

Comment:
Since each of these error conditions are being checked individually, could the error reporting be more specific.  Knowing which of these several conditions fails is useful information.

NULL_STORE
NULL_NAME_MAP